pkgs/by-name/re/renode-unstable-bin/package.nix

```diff
@@ -1,14 +1,13 @@
 {
-  renode,
   fetchurl,
+  renode-bin,
   writeScript,
 }:
 
-renode.overrideAttrs (
+renode-bin.overrideAttrs (
   finalAttrs: _: {
     pname = "renode-unstable";
-    version = "1.15.3+20250711gitb35bde0fb";
-
+    version = "1.15.3+20250707gita02ab2a10";
```

Comment:
I think that you are changing the version by mistake, so you shouldn't do that. Please preserve the original one while renaming the recipe.